### PR TITLE
Added docs and embed option.

### PR DESCRIPTION
**Pull request by jhennawi**
This improves the documentation on pypeit_view_fits which was unclear for mosaics. It also enables embedding in the code, which is quite useful when checking gratings tilts at the telescope.

### CODE REVIEW

File: pypeit/scripts/view_fits.py [hunk 1]
```diff
@@ -33,11 +33,14 @@ def get_parser(cls, width=None):
                             help='Show a FITS extension in the raw file. Note --proc and --mosaic '
                                  'will not work with this option.')
         parser.add_argument('--det', type=str, default='1', nargs='*',
-                            help='Detector(s) to show.  If more than one, the list of detectors '
-                                 'must be one of the allowed mosaics hard-coded for the selected '
+                            help='Detector(s) to show.  If more than one, the list of detectors, i.e. --det 4 8 '
```
Comment (profxj) [inline pypeit/scripts/view_fits.py:37]: is it `4 8` or `4,8` or are both allowed?

File: pypeit/scripts/view_fits.py [hunk 2]
```diff
@@ -126,3 +129,6 @@ def main(args):
         display.show_image(img, chname=args.chname)
 
 
+        if args.embed:
+            embed()
```
Comment (profxj) [inline pypeit/scripts/view_fits.py]: can you add  `header='line 133 of view_fits.py`
  so people know where they landed?

**Review by profxj — APPROVED** [review]
two minor things so I am approving

File: pypeit/scripts/view_fits.py (hunk 2 above)
Comment (kbwestfall) [reply pypeit/scripts/view_fits.py]: I would instead use `pypeit.utils.embed_header()`.  It automatically pulls this info from the stack so it's always correct.

File: pypeit/scripts/view_fits.py (hunk 1 above)
Comment (kbwestfall) [reply pypeit/scripts/view_fits.py:37]: Good question.  It actually looks like it has to be '4 8', and '4,8' will likely barf.  We should think about changing this.

**Review by kbwestfall — APPROVED** [review]
Looks good!  I had a couple of follow-ups based on @profxj 's comments.

File: pypeit/scripts/view_fits.py (hunk 1 above)
Comment (jhennawi) [reply pypeit/scripts/view_fits.py:37]: Yeah, I think MSC01 should also be allowed, which it is not.

File: pypeit/scripts/view_fits.py (hunk 2 above)
Comment (jhennawi) [reply pypeit/scripts/view_fits.py]: Done and made the same changed to pypeit_show_2dspec which also has an embed option.

**Pull request merged**